pyscisci/embedding.py

```diff
@@ -0,0 +1,437 @@
+# -*- coding: utf-8 -*-
+"""
+.. module:: embedding
+    :synopsis: The main Embedding Class
+
+.. moduleauthor:: Jisung Yoon <jisung.yoon92@gmail.com>
+ """
+import sys
+import numpy as np
+from scipy import sparse
+import networkx as nx
+import json
+from joblib import Parallel, delayed
+from scipy import spatial
+
+
+# For now, I dont think we need to make the full pySciSci package dependent on these packages
+try:
+    import numba
+except ImportError:
+    raise ImportError('Please install numba to take full advantage of fast embedding tools. \n pip install numba')
+    
+try:    
+    from gensim.models import Word2Vec
+except ImportError:
+    raise ImportError('Please install gensim to take full advantage of fast embedding tools. \n pip install gensim')
+
+# determine if we are loading from a jupyter notebook (to make pretty progress bars)
+if 'ipykernel' in sys.modules:
+    from tqdm.notebook import tqdm
+else:
+    from tqdm import tqdm
+    
+
+
+
+    
+# Node2Vec instaces    
+class Node2Vec(object):
```

Comment:
Can you edit the class description to give a very brief overview of the method (ie that you first sample tons of random walks on the network, and then you embed them using Word2Vec).